mergify_engine/actions/review.py

```diff
@@ -39,13 +40,42 @@ class ReviewAction(actions.Action):
         voluptuous.Required("message", default=None): voluptuous.Any(
             types.Jinja2, None
         ),
+        voluptuous.Required("review_bot_account", default=None): voluptuous.Any(
```

Comment:
```suggestion
        voluptuous.Required("bot_account", default=None): voluptuous.Any(
```
We only prefixed it in the merge action because it'd do multiple "actions", but here there's only one.